examples/us-dams/README.md

```diff
@@ -0,0 +1,53 @@
+# US dams
+
+This is an [Observable Framework](https://observablehq.com/framework) project. To start the local preview server, run:
```

Comment:
Can we use README.md to explain what R packages need to be installed (and how?).

fwiw I had to run the following command (which took ~15 minutes):

```
R
R> install.packages("tidyverse")
```


It would also be a good place to highlight data provenance (I see you have some HTML comments in the md files)?